src/pe_source/intelx_identity.py

```diff
@@ -0,0 +1,289 @@
+
+        leaks_json = self.find_credential_leaks(
+            roots_df["root_domain"].values.tolist(), START_DATE, END_DATE
+        )
+        if len(leaks_json) < 1:
+            LOGGER.info(f"No credentials found for {cyhy_org_id}")
+            return 0
+        creds_df, breaches_df = self.process_leaks_results(leaks_json, pe_org_uid)
+        # Insert breach data into the PE database
+        try:
+            insert_intelx_breaches(breaches_df)
+        except Exception as e:
+            LOGGER.error("Failed inserting breaches for %s", cyhy_org_id)
+            LOGGER.error(e)
+            return 1
+
+        breach_dict = get_intelx_breaches(source_uid)
+        breach_dict = dict(breach_dict)
+        for cred_index, cred_row in creds_df.iterrows():
+            breach_uid = breach_dict[cred_row["breach_name"]]
+            creds_df.at[cred_index, "credential_breaches_uid"] = breach_uid
+        try:
+            insert_intelx_credentials(creds_df)
+        except Exception as e:
+            LOGGER.error("Failed inserting credentials for %s", cyhy_org_id)
+            LOGGER.error(e)
+            return 1
+        return 0
+
+    def query_identity_api(self, domain, start_date, end_date):
+        """Create an initial search and return the search id."""
+        url = f"https://3.intelx.io/accounts/csv?selector={domain}&k={api_key}&datefrom={start_date}&dateto={end_date}"
+        payload = {}
+        headers = {}
+        attempts = 0
+        while attempts < 5:
+            try:
+                response = requests.request("GET", url, headers=headers, data=payload)
+                break
+            except requests.exceptions.Timeout:
+                time.sleep(5)
+                attempts += 1
+                if attempts == 5:
+                    LOGGER.error("IntelX identity is not responding. Exiting program.")
+                    sys.exit()
+                LOGGER.info("IntelX Identity API response timed out. Trying again.")
+            except Exception as e:
+                LOGGER.error(f"Error occured geting search id: {e}")
+                return 0
+        LOGGER.info("Acquired search id.")
+        time.sleep(5)
+        return response.json()
+
+    def get_search_results(self, id):
+        """Search IntelX for email leaks."""
+        url = f"https://3.intelx.io/live/search/result?id={id}&format=1&k={api_key}"
+
+        payload = {}
+        headers = {}
+        attempts = 0
+        try:
+            response = requests.request("GET", url, headers=headers, data=payload)
+        except requests.exceptions.Timeout:
+            time.sleep(5)
+            attempts += 1
+            if attempts == 5:
+                LOGGER.error("IntelX identity is not responding. Exiting program.")
+                sys.exit()
+            LOGGER.info("IntelX Identity API response timed out. Trying again.")
+        except Exception as e:
+            LOGGER.error(f"Error occured geting search results: {e}")
```

Comment:
Fix spelling errors:
```suggestion
            LOGGER.error(f"Error occurred getting search results: {e}")
```